packages/aws-cdk/bin/cdk.ts

```diff
@@ -60,6 +60,7 @@ async function parseCommandLineArguments() {
     .command('destroy [STACKS..]', 'Destroy the stack(s) named STACKS', yargs => yargs
       .option('force', { type: 'boolean', alias: 'f', desc: 'Do not ask for confirmation before destroying the stacks' }))
     .command('diff [STACK]', 'Compares the specified stack with the deployed stack or a local template file, and returns with status 1 if any difference is found', yargs => yargs
+      .option('json-context', { type: 'number', desc: 'number of context lines to include in arbitrary JSON diff rendering', default: 5 })
```

Comment:
I think `context` should be fine here with the description

Reply:
Except if collides with the "global" `--context` that is used to pass context key-value pairs to the CDK app...

Reply:
I'm still open if you have a better suggestion for a name (I don't love this one, but I find `json-diff-context` too verbose).